my_game.py

```diff
@@ -194,13 +195,12 @@ def on_update(self, delta_time):
 
         # Move player with keyboard
         if self.left_pressed and not self.right_pressed:
-            self.player_sprite.change_x = -PLAYER_SPEED_X
+            self.player_sprite.angle+= PLAYER_ROTATE_SPEED
         elif self.right_pressed and not self.left_pressed:
-            self.player_sprite.change_x = PLAYER_SPEED_X
-
+            self.player_sprite.angle+= -PLAYER_ROTATE_SPEED
         # Move player with joystick if present
         if self.joystick:
-            self.player_sprite.change_x = round(self.joystick.x) * PLAYER_SPEED_X
+            self.player_sprite.change_x = round(self.joystick.x) * PLAYER_ROTATE_SPEED
```

Comment:
?? I want my PLAYER_SPEED_X back... PLAYER_ROTATE_SPEED should have no impact in change_x.